package.json

```diff
@@ -8,7 +8,7 @@
     "email": "me@chadalen.com",
     "url": "https://cadamsdev.com"
   },
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
You don't have to revert this change but just FYI you don't need to update this. I have semantic-release setup which should handle creating the new version automatically as long as there is a valid commit message. (see [here](https://github.com/cadamsdev/vscode-jetbrains-icon-theme/blob/main/docs/CONTRIBUTING.md#commit-messages))

Reply:
Ah, okay, got it.